src/js/structs/Volume.js

```diff
@@ -0,0 +1,36 @@
+import Item from './Item';
+import VolumeStatus from '../constants/VolumeStatus';
+
+class Volume extends Item {
+
+  getContainerPath() {
```

Comment:
We can't use proper getters yet, as they interfere with the way our `Item` struct is working. Let's add this to our agenda and schedule some time to fix the item struct.